### PR TITLE
Add squad requests

### PR DESCRIPTION
**Pull request by ARtoriouSs**
This branch is above seeds branch from #10, so it contains its changes. It should me rebased on master after #10 merge

### CODE REVIEW

**Comment by mirterious** [comment]
> This branch is above seeds branch from #10, so it contains its changes. It should me rebased on master after #10 merge

Rebase, please

File: lib/squadster/formations/squad_member.ex [hunk 1]
```diff
@@ -10,6 +10,7 @@ defmodule Squadster.Formations.SquadMember do
     field :role, RoleEnum
     belongs_to :user, Squadster.Accounts.User
     belongs_to :squad, Squadster.Formations.Squad
+    has_many :approved_squad_requests, Squadster.Formations.SquadRequest, foreign_key: :approver_id
```
Comment (mirterious) [inline lib/squadster/formations/squad_member.ex]: why there is `many`, correct me if i am wrong, but one **member** can hold only one **approved_squad_request**

Comment (mirterious) [reply lib/squadster/formations/squad_member.ex]: maybe we should make `disapproved_squad_requests`, but in my o3.14nion :) it doesn't make any sence

Comment (mirterious) [reply lib/squadster/formations/squad_member.ex]: please, share you thoughts about this

**Review by mirterious — CHANGES_REQUESTED** [review]

File: lib/squadster/formations/squad_member.ex (hunk 1 above)
Comment (ARtoriouSs) [reply lib/squadster/formations/squad_member.ex]: One member can hold multiple requests that was **approved by him**, e.g. commander/deputy can approve all the requests, and `approved_squad_requests` will return all of them. Not sure about disapproved, I think we can just delete request if it was declined. Also maybe this field isn't really needed, maybe we can leave only `approver` on request, but they don't load automatically and don't have any performance impact, so why not?

Comment (ARtoriouSs) [reply lib/squadster/formations/squad_member.ex]: Removed ✔️

**Review by mirterious — APPROVED** [review]

**Pull request merged**